### ISSUE TITLE
Feature Request: Limit the options for "Change Color"

### ISSUE THREAD

**Issue by AtsusaKaneytza** [mypaint/mypaint#319]
This is the menu option/shortcut in question:
![image](https://cloud.githubusercontent.com/assets/8382492/7737247/4bd1bccc-ff0f-11e4-9253-3363fc9a9373.png)

It's very useful for quickly picking colors while speedpainting; I'm used to using the concentric circles when changing colors. I discovered (the hard way) that it had been updated with all of the color pickers available in the program.

Which is neat! Except when you accidentally skip the color picker you wanted to use, you have to cycle past
![image](https://cloud.githubusercontent.com/assets/8382492/7737238/406344aa-ff0f-11e4-90ae-b484a23ed08b.png)

Which makes things take longer than originally intended when you just wanted to change the color... I see there is also the "Fast Subset" option, but again, it's not really customizable/doesn't have the pickers I'd normally like to use.

So basically, what I'm asking is that maybe in User Preferences or something, the user can customize which color pickers/color changers they prefer to use concerning the "Change Color" and "Fast Subset" menu options.

**Comment by achadwick** [comment]
Just to clarify, is the accidental skip-past happening when you bounce on the "C" (or "V", or "B"), or are you using the arrows at the top of the popups themselves?

![ears](https://cloud.githubusercontent.com/assets/61299/7757812/e99edcee-fffc-11e4-9f50-88c4ca191ceb.png)

**Comment by AtsusaKaneytza** [comment]
Both.

**Comment by achadwick** [comment]
Just reviewing issues.

@AtsusaKaneytza can you go into a bit more detail about the accidental skip-past you've been experiencing? I want to rule out this being a bug (a buggy double-or-more forward through the popup in response to a *single* input event), so that we can address it as a design shortcoming. That's the impression I've got so far, anyway :smiley: 

Deferring until after the 1.2.0 release, since this cannot be addressed with extra preferences now that strings are frozen.

**Comment by AtsusaKaneytza** [comment]
It's not a bug, it's just me being too click/keystroke happy and wanting to move fast. It's not an actual issue, which is why I said "feature request."

**Comment by odysseywestra** [comment]
@AtsusaKaneytza Could you move this thread over to https://community.mypaint.org/c/feature-requests? I'm working on cleaning up the issue tracker by move all feature requests there so we can get community feedback. If you can also set your post as a wiki that way we can edit the scope of the feature request that would be awesome as well. Don't forget to link back to here as well so I can keep track of it incase we need to reopen the issue. Thanks.
